tests/unit/test_app.py

```diff
@@ -77,6 +78,49 @@ def name(name):
     return demo
 
 
+def test_mixed_content_types():
+    demo = app.Chalice('demo-app')
+
+    with pytest.raises(ValueError):
+        @demo.route('/index', content_types=['application/octet-stream',
+                                         'text/plain'])
```

Comment:
Hmm, I'm guessing we're not running flake8 on our tests...we should probably get that updated (as a separate PR).